web/src/containers/StateDashboardSidebar.test.js

```diff
@@ -33,8 +33,8 @@ describe('State dashboard sidebar component', () => {
   });
 
   test('selects an apd', () => {
-    const component = shallow(<Sidebar {...props} />);
-    component.find('SidebarLink[hash="1"]').simulate('click');
+    const component = mount(<Sidebar {...props} />);
+    component.find('VerticalNavItem[id="1"]').simulate('click');
     expect(props.selectApd.calledWith(1)).toBeTruthy();
```

Comment:
This is failing with a return value of `false`. I'm not entirely sure we need to be testing this, now that it's not a direct child of `StateDashboardSidebar`, but it does seem like we should be testing `selectApd` somehow? @mgwalker, I'd love your opinions on this.